src/cloud/utils.js

```diff
@@ -83,6 +138,55 @@ export async function getTerraformPrompt() {
   });
 }
 
+export async function getSecretSubCommandPrompt() {
+  const secretCommands = ['get', 'set'];
+  return await prompt({
+    type: 'select',
+    message: 'Select "get" or "set" secret:',
+    choices: secretCommands.map((value) => {
+      return { title: value, value };
+    }),
+  });
+}
+
+export async function getSecretNamePrompt() {
+  return await prompt({
+    type: 'text',
+    message: 'Enter secret name:',
+    initial: 'credentials',
+    validate: (value) =>
+      value.replace(/[^a-z0-9_-]/gim, '') != value
```

Comment:
could just be a `.match` here no?

Reply:
Yes, `!value.match(/[^a-z0-9_-]/gim)` works too.